features-json/colr.json

```
@@ -508,9 +508,9 @@
       "16.4":"y",
       "16.5":"y",
       "16.6":"y",
-      "17.0":"y",
-      "17.1":"y",
-      "17.2":"y"
+      "17.0":"n #4",
+      "17.1":"n",
+      "17.2":"n"
```

I think it makes sense to put the note on all versions starting 17, assuming it's not intended to be removed.
If it turns out to be intended I guess just `n` it is.
But for now:

```suggestion
      "17.0":"n #4",
      "17.1":"n #4",
      "17.2":"n #4"
```